src/quantum/azext_quantum/operations/job.py

```diff
@@ -0,0 +1,238 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=redefined-builtin
+
+import logging
+
+from knack.util import CLIError
+
+from .._client_factory import cf_jobs, _get_data_credentials, base_url
+from .workspace import WorkspaceInfo
+from .target import TargetInfo
+
+logger = logging.getLogger(__name__)
+
+
+def list(cmd, resource_group_name=None, workspace_name=None):
```

Comment:
If `resource_group_name` and `workspace_name` are mandatory, we can just remove the default value `=None`. The framework will define it as required and do the validation automatically.

Reply:
No because they are not necessarily required, the user has the option to specify a default target and default workspace using `az quantum target set` and `az quantum workspace set` accordingly, if they do it is saved as config settings, as such I believe we do need the custom validators.